docs/quest_template.md

```diff
@@ -851,8 +848,7 @@ Time in seconds that the player has to complete this quest.
 
 ### ~~RequiredClasses~~
```

Comment:
Does this field exist in AC?

Reply:
I don't see it in the table but that surprises me

Reply:
So remove it from wiki?